site/docs/src/json/identity-providers/search-response.json

```diff
@@ -0,0 +1,22 @@
+{
+  "identityProviders": [
+    {
+      "applicationConfiguration": {
+        "1c212e59-0d0e-6b1a-ad48-f4f92793be32": {
+          "createRegistration": true,
+          "enabled": true
+        }
+      },
+      "buttonText": "Login with Twitter",
+      "consumerKey": "24iuleLRKIZUNhxuuUK6yDZwb",
+      "consumerSecret": "e2ZpAaAfxv2j9eeh6JTeNcXMWdVLjRNriXm4wSZt1f1Ss3Syp4",
+      "enabled": true,
+      "id": "45bb233c-0901-4236-b5ca-ac46e2e0a5a5",
+      "insertInstant": 1595361142909,
+      "lastUpdateInstant": 1595361143101,
+      "name": "Twitter",
+      "type": "Twitter"
+    }
+  ],
+  "total": 2
```

Comment:
Is this 2 because you specified a limit of 1 in the query? I find that confusing (similar to my comment here: https://github.com/FusionAuth/fusionauth-site/pull/1903#discussion_r1098977226 ).